eoscompanion/ekn_query.py

```diff
@@ -68,7 +68,7 @@ def _on_received_application_info(_, result):
                 'title': model['title'],
                 'contentType': 'application/x-ekncontent-set',
                 'thumbnail': format_thumbnail_uri(application_id,
-                                                  model,
+                                                  model.get('thumbnail_uri', None),
```

Comment:
Prefer to use the `-` version of properties (eg, `model.get('thumbnail-uri', None)`